src/app/accounts/PageAccounts.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Alert,
+  AlertIcon,
+  Button,
+  Flex,
+  Heading,
+  Stack,
+  StackDivider,
+  Text,
+} from '@chakra-ui/react';
+import { signIn } from 'next-auth/react';
+import { useTranslation } from 'react-i18next';
+import { FiGithub, FiGitlab } from 'react-icons/fi';
+
+import { Page, PageContent } from '@/app/layout';
+import { Icon } from '@/components';
+import { trpc } from '@/utils/trpc';
+
+export const PageAccounts = () => {
+  const { t } = useTranslation('account');
+
+  const { data, isLoading: areAccountsLoading } = trpc.useQuery(['account.me']);
+
+  const githubAccount = data?.find((account) => account.provider === 'github');
+  const gitlabAccount = data?.find((account) => account.provider === 'gitlab');
```

Comment:
This can probably be set in the backend to avoid client side computation and enjoy tRPC typings.